controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -371,3 +374,21 @@ func (r *DataScienceClusterReconciler) watchDataScienceClusterResources(a client
 		return nil
 	}
 }
+
```

Comment:
Should this be removed @zdtsw ?

Reply:
ah, the comments, yes, i will clean it up.